File: nastenka/check_users.php
<?php
require_once __DIR__ . '/config.php';
$pdo = new PDO("mysql:host=db1.usr.sk;dbname=vueto;charset=utf8mb4", "vueto.sk", "KHfgSjbar(819nNE");
$users = $pdo->query("SELECT id, username, full_name, email, role, is_active FROM crm_users")->fetchAll(PDO::FETCH_ASSOC);
echo json_encode($users, JSON_PRETTY_PRINT);

File: nastenka/check_users.py
import pymysql
import json

try:
    connection = pymysql.connect(
        host='db1.usr.sk',
        user='vueto.sk',
        password='KHfgSjbar(819nNE',
        database='vueto',
        cursorclass=pymysql.cursors.DictCursor
    )
    with connection.cursor() as cursor:
        cursor.execute("SELECT id, username, full_name, email, role, is_active FROM crm_users")
        users = cursor.fetchall()
        print(json.dumps(users, indent=2))
except Exception as e:
    print("Error:", e)
finally:
    if 'connection' in locals():
        connection.close()
